monkey/monkey_island/cc/environment/environment_config.py

```diff
@@ -37,22 +39,29 @@ def _load_from_json(self, config_json: str) -> EnvironmentConfig:
 
     def _load_from_dict(self, dict_data: Dict):
         user_creds = UserCreds.get_from_dict(dict_data)
-        aws = dict_data['aws'] if 'aws' in dict_data else None
+        aws = dict_data["aws"] if "aws" in dict_data else None
+        data_dir = (
+            dict_data["data_dir"] if "data_dir" in dict_data else DEFAULT_DATA_DIR
+        )
 
-        self.server_config = dict_data['server_config']
-        self.deployment = dict_data['deployment']
+        self.server_config = dict_data["server_config"]
+        self.deployment = dict_data["deployment"]
         self.user_creds = user_creds
         self.aws = aws
+        self.data_dir = data_dir
 
     def save_to_file(self):
-        with open(self._server_config_path, 'w') as f:
+        with open(self._server_config_path, "w") as f:
             f.write(json.dumps(self.to_dict(), indent=2))
 
     def to_dict(self) -> Dict:
-        config_dict = {'server_config': self.server_config,
-                       'deployment': self.deployment}
+        config_dict = {
+            "server_config": self.server_config,
+            "deployment": self.deployment,
+            "data_dir": self.data_dir,
```

Comment:
Same complaint, this should be more specific. In the context of environment config, this could be something like `path_to_dir_of_config_files` or something.